server/tests/service/triviaService.test.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable no-unused-vars */
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+
+import TriviaService from '../../src/service/triviaService';
+
+describe('TriviaService', () => {
+  const validConfig = { baseUrl: 'http://example.org' };
+
+  describe('#constructor - config validation', () => {
+    it('valid config, no exception', () => {
+      expect(() => {
+        const triviaService = new TriviaService(validConfig);
+      }).not.toThrow();
+    });
+    it('throws exception on invalid config', () => {
+      expect(() => {
+        const triviaService = new TriviaService({});
+      }).toThrow();
+    });
+  });
+
+  describe('#buildTriviaUrl', () => {
+    let triviaService = null;
+
+    beforeEach(() => {
+      triviaService = new TriviaService(validConfig);
+    });
+
+    it('builds correct url', () => {
+      const today = new Date('2017-09-06');
+      const url = triviaService.buildTriviaUrl(today);
+      const expectedUrl = 'http://example.org/9/6/date';
+      expect(url).toEqual(expectedUrl);
+    });
+
+    it('builds correct url with invalid input', () => {
+      const today = new Date();
+      const url = triviaService.buildTriviaUrl('invalid');
+      const expectedUrl = triviaService.buildTriviaUrl(today);
+      expect(url).toEqual(expectedUrl);
```

Comment:
What is this supposed to test? Passing a date into a buildUrl function?

Reply:
It is to test if you pass invalid date to buildTriviaUrl, it will build the url for today's date.